data_model/src/numeric.rs

```diff
@@ -310,7 +306,7 @@ mod tests {
             NumericValue::Fixed((u32::MAX as f64).try_into().expect("trivial conversion")),
         ];
         for val in values {
-            let new_value: NumericValue = format!("{val}").parse().expect("Failed to parse");
+            let new_value: NumericValue = format!("{val:?}").parse().expect("Failed to parse");
```

Comment:
Why did you change this? Does the `Display` implementation not output the suffix?

Reply:
Serialization was based on Debug repetition from the beginning, i’m not sure why Display was used in the first place. 
And, yes, afaik display doesn’t print suffixes.